scene/gui/grid_container.cpp

```diff
@@ -46,7 +46,7 @@ void GridContainer::_notification(int p_what) {
 			int hsep = get_constant("hseparation");
 			int vsep = get_constant("vseparation");
 			int max_col = MIN(get_child_count(), columns);
-			int max_row = get_child_count() / columns;
+			int max_row = ceil((float)get_child_count() / (float)columns);
```

Comment:
Based on a quick search through the repo I am pretty sure you should be using static_cast<float> instead of C style casts.

Reply:
Actually we do tend to favor C style casts for atomic data types.

Reply:
```
$ rg -g'!thirdparty' "static_cast<float>" | wc -l
5
$ rg -g'!thirdparty' "\(float\)" | wc -l
315
```

Reply:
very well, consider me corrected 👍

Reply:
We actually should choose to stick with one or the other.

Reply:
@YeldhamDev We should also decide on whether to use `int(value)` or `(int)value`, as both forms are valid in Godot's codebase.